pubspec.yaml

```
@@ -1,7 +1,7 @@
 name: ardrive_ui
 description: UI Design Library for the ArDrive Design System
 
-version: 1.8.0
+version: 1.8.1
```

The public API of `ArDriveHoverWidget` is changing. Shouldn't it be v1.9.0?
```suggestion
version: 1.9.0
```